app/src/main/java/com/babylon/wallet/android/presentation/ui/composables/assets/NFTsTab.kt

```diff
@@ -146,13 +146,7 @@ private fun NFTHeader(
                 }
 
                 Text(
-                    collection.currentSupply?.let { currentSupply ->
-                        stringResource(
-                            id = R.string.assetDetails_NFTDetails_ownedOfTotal,
-                            collection.items.size,
-                            currentSupply
-                        )
-                    } ?: collection.amount.toString(),
+                    text = collection.amount.toString(),
```

Comment:
I am not sure if the iOS look and feel is the correct one. I remember us implementing the spec as it is and for iOS there was a tech limitation for not being implemented.

Reply:
Agreed, this is a quick and safe (compromise) solution for this release. We agreed to get back to this later and make it consistent across platforms. If you feel like it can be properly fixed now though, that would be very nice!

Reply:
I'll merge it to unblock the testing and the release.

Reply:
Ah got it, didn't read through the thread. Thanks for the clarification @sergiupuhalschi-rdx !